### PR TITLE
My Home: Add Store education card

### PR DESCRIPTION
**Pull request by sixhours**
#### Changes proposed in this Pull Request

* Add a new card component for the store for launched sites with the `sell` intent
* Must be run with D75890-code and the API sandboxed

**Visual demo**

Screen Shot 2022-03-02 at 3 53 36 PM

#### Testing instructions

* Switch to this PR, navigate to `/home` on a launched site that's created through the seller experience flow
* Apply D75890-code to your sandbox and sandbox `public-api.wordpress.com`
* You should see the new "How to set up an online store" card in the educational section of My Home.

Related to #61467

### CODE REVIEW

**Comment by blackjackkent** [comment]
Behaviorally this looks great - however, the requirements for the issue state we need to display this specifically for users with the "sell" intent. It doesn't look like we're checking for that right now; is that intended for a future PR?

**Comment by sixhours** [comment]
> Behaviorally this looks great - however, the requirements for the issue state we need to display this specifically for users with the "sell" intent. It doesn't look like we're checking for that right now; is that intended for a future PR?

Aha, I missed that! I will fix that, as well as updating the illustration to use the correct one :)

**Comment by sixhours** [comment]
@blackjackkent I updated the server-side diff to add a check for the intent before displaying the card!

File: client/my-sites/customer-home/cards/education/store/index.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	return (
+		<EducationalContent
+			title={ translate( 'How to set up an online store' ) }
+			description={ translate(
+				'Payments features on WordPress.com allow you to make money from your site in many ways.'
+			) }
+			links={ [
+				{
+					externalLink: true,
+					url: `https://wordpress.com/support/video-tutorials-add-payments-features-to-your-site-with-our-guides/`,
```
Comment (sixhours) [inline client/my-sites/customer-home/cards/education/store/index.jsx:19]: This may need `localizeUrl`, need to double check.

Comment (sixhours) [reply client/my-sites/customer-home/cards/education/store/index.jsx:19]: It looks like this content hasn't been translated to other languages yet.

**Comment by andres-blanco** [comment]
added a comment on D75890-code

**Review by andres-blanco — APPROVED** [review]
LGTM! Remember to merge this AFTER D75890-code

image

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7171556

Thank you @sixhours for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7171556).

**Pull request merged**